lms/djangoapps/onboarding/views.py

```diff
@@ -469,11 +470,20 @@ def update_account_settings(request):
 
     _, unattended_org_surveys = affiliated_unattended_surveys(user_extended_profile)
 
+    pending_new_email = ''
+    is_email_change_pending = False
+    email_change_pending = PendingEmailChange.objects.filter(user=request.user).first()
```

Comment:
`email_change_pending` feels that this variable contains a boolean. How about changing it to `pending_email_change`, if that makes more sense to you as well?